cmd/changes_get_change.go

```diff
@@ -43,6 +43,11 @@ const assetVersion = "17c7fd2c365d4f4cdd8e414ca5148f825fa4febd"
 func GetChange(cmd *cobra.Command, args []string) error {
 	ctx := cmd.Context()
 
+	app, err := getAppUrl(viper.GetString("frontend"), viper.GetString("app"))
+	if err != nil {
+		log.Fatalf("Error: %v", err)
```

Comment:
Never use `log.Fatal`, as that will `os.Exit()` without finishing tracing and other cleanup tasks.